dashboard/app/assets/javascripts/editor/json_editor.js

```diff
@@ -0,0 +1,106 @@
+/* global CodeMirror */
+
+function calloutEditor(container, options) {
+
```

Comment:
There are a couple other mentions of callouts in this file as well. Not sure if any are intentional.